python/ql/lib/semmle/python/frameworks/Aiomysql.qll

```diff
@@ -15,15 +15,15 @@ private module Aiomysql {
   private import semmle.python.internal.Awaited
 
   /**
-   * A `ConectionPool` is created when the result of `aiomysql.create_pool()` is awaited.
+   * Gets the `ConnectionPool` that is created when the result of `aiomysql.create_pool()` is awaited.
```

Comment:
```suggestion
   * Gets a `ConnectionPool` that is created when the result of `aiomysql.create_pool()` is awaited.
```
This being an API node, I think we should express that multiple results are likely.